ui/components/src/TimeChart/TimeChart.tsx

```diff
@@ -0,0 +1,364 @@
+// Copyright 2023 The Perses Authors
```

Comment:
Nit: `TimeChart` -> `TimeSeriesChart`

Reply:
I kind of like TimeChart for now, since there are various exports of TimeSeriesChart already from panels-plugin and internally the forked LineChart is called TimeSeriesChart, maybe after we consolidate or figure out the plan for a context-aware TimeChart, we can rename it (possibly TimeSeriesViz right?)